server/routes.go

```diff
@@ -1343,6 +1348,94 @@ func (s *Server) PsHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, api.ProcessResponse{Models: models})
 }
 
+func (s *Server) InfoHandler(c *gin.Context) {
+	runnerMap := runners.GetAvailableServers(s.runnerDir)
+	runners := []string{}
+	for k := range runnerMap {
+		runners = append(runners, k)
+	}
+	sort.Strings(runners)
+
+	sysInfo := discover.GetSystemInfo()
+	ms, _ := Manifests()
+	fsUsed := uint64(0)
+	layers := map[string]int64{}
+	for _, m := range ms {
+		for _, l := range m.Layers {
+			layers[l.Digest] = l.Size
+		}
+	}
+	for _, sz := range layers {
```

Comment:
It looks like you can get rid of this loop and just add to `fsUsed` when you're adding `l.Size` above.